app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+import os
+
+
+class DirectoryExistError(Exception):
+
+    def __init__(self, path: str) -> None:
+        self.path = path
+
+    def __str__(self) -> str:
+        return f"Directory with path {self.path} is already exist!"
```

Comment:
Use built-in exception https://docs.python.org/3/library/exceptions.html#FileExistsError